modules/apps/foundation/configuration-admin/configuration-admin-test/src/testIntegration/java/com/liferay/configuration/admin/internal/util/test/ConfigurationDDMFormDeclarationUtilTest.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.internal.util.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.configuration.admin.definition.ConfigurationDDMFormDeclaration;
+import com.liferay.configuration.admin.web.internal.util.ConfigurationDDMFormDeclarationUtil;
+import com.liferay.portal.kernel.util.StringPool;
+
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceReference;
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+
+/**
+ * @author Pei-Jung Lan
+ */
+@RunWith(Arquillian.class)
+public class ConfigurationDDMFormDeclarationUtilTest {
+
+	@Before
+	public void setUp() throws Exception {
+		_bundle = FrameworkUtil.getBundle(
+			ConfigurationDDMFormDeclarationUtilTest.class);
+
+		_bundleContext = _bundle.getBundleContext();
+
+		ServiceReference<ConfigurationAdmin> serviceReference =
+			_bundleContext.getServiceReference(ConfigurationAdmin.class);
+
+		ConfigurationAdmin configurationAdmin = _bundleContext.getService(
+			serviceReference);
+
+		_configuration = configurationAdmin.createFactoryConfiguration(
+			"test.pid", StringPool.QUESTION);
```

Comment:
Let's use the `OSGiServiceUtil`.  You can see an example of this in the constructor for `ConfigurationTemporarySwapper`.